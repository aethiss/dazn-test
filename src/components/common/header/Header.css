.site-header {
  display: flex;
  flex-direction: row;
  justify-content: flex-end;
  background-color: #000000;
}

.header-right {
  width: 19%;
}

.header-middle {
  width: 79%;
  justify-self: center;
  align-self: center;
}

.header-logo {
  max-height: 55px;
}

.header-searchbox {
  /*position: relative; !* or absolute *!*/
  /*top: 50%;*/
  /*left: 40%;*/

}
