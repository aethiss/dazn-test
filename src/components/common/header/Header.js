import React from 'react'

import style from "./Header.css"

const Header = () => (
  <header className="site-header" role="navigation">
    <div className="header-right">
      <img className="header-logo" src="../../../../assets/images/dazn-logo.png" />
    </div>
  </header>
)

export default Header
