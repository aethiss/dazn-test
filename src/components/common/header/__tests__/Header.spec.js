import React from 'react'
import { shallow } from 'enzyme'

import Header from '../Header'

describe('@Header', () => {
  describe('render', () => {
    it('should render defaultheader', () => {
      const component = shallow(<Header />)
      expect(component).toMatchSnapshot()
    })
  })
})
