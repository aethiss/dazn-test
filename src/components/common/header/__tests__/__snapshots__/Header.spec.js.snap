// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@Header render should render defaultheader 1`] = `
<header
  className="site-header"
  role="navigation"
>
  <div
    className="header-right"
  >
    <img
      className="header-logo"
      src="../../../../assets/images/dazn-logo.png"
    />
  </div>
</header>
`;
